docs/evaluation/simulate.rst

```diff
@@ -106,20 +170,41 @@ slightly different answers. With default settings, the input model is
 convolved with the RMF and multiplied by the ARF, and properly scaled
 for the exposure time.
 
-  >>> from sherpa.astro.fake import fake_pha
-  >>> fake_pha(data, srcmdl)
+::
+
+   >>> from sherpa.astro.fake import fake_pha
+   >>> from sherpa.astro.plot import DataPHAPlot
+   >>> fake_pha(data, instmdl)  # NOTE: include the instrument response
```

Comment:
We have this text above about "running twice will give different answers". Should we tie that back to how the example above used the rng setting to avoid that? e.g. we could say "The example above set the seed of a random number generator "rng" and passed that into the "poisson noise" function. Here, we do not pass anything in and thus we get different answers".
I'm not sure it's worht the effort. Most people want just some spectrum to show for a proposal and don't care if it's slightly different next time, but if we think that's an important point, we should get back to it here. I'm not sure what is moportant for most users here.